src/Symfony/Component/VarDumper/Tests/Cloner/VarClonerTest.php

```diff
@@ -379,6 +382,59 @@ public function testJsonCast()
         $this->assertStringMatchesFormat(\PHP_VERSION_ID >= 70200 ? str_replace('"1"', '1', $expected) : $expected, ob_get_clean());
     }
 
+    public function testGmpCast()
+    {
+        if (false === extension_loaded('gmp')) {
+            $this->markTestSkipped('GMP PHP extension should be loaded to be able to parse GMP resources');
+        }
+        $data = gmp_init('0101');
+        $cloner = new VarCloner();
+
+        $clone = $cloner->cloneVar($data);
+        $expected = <<<'EODUMP'
+"""
+Symfony\Component\VarDumper\Cloner\Data Object\n
+(\n
+    [data:Symfony\Component\VarDumper\Cloner\Data:private] => Array\n
+        (\n
+            [0] => Array\n
+                (\n
+                    [0] => Symfony\Component\VarDumper\Cloner\Stub Object\n
+                        (\n
+                            [type] => 4\n
+                            [class] => GMP\n
+                            [value] => \n
+                            [cut] => 0\n
+                            [handle] => 31\n
+                            [refCount] => 0\n
+                            [position] => 1\n
+                            [attr] => Array\n
+                                (\n
+                                )\n
+\n
+                        )\n
+\n
+                )\n
+\n
+            [1] => Array\n
+                (\n
+                    [\x00~\x00gmpString] => 65\n
+                )\n
+\n
+        )\n
+\n
+    [position:Symfony\Component\VarDumper\Cloner\Data:private] => 0\n
+    [key:Symfony\Component\VarDumper\Cloner\Data:private] => 0\n
+    [maxDepth:Symfony\Component\VarDumper\Cloner\Data:private] => 20\n
+    [maxItemsPerDepth:Symfony\Component\VarDumper\Cloner\Data:private] => -1\n
+    [useRefHandles:Symfony\Component\VarDumper\Cloner\Data:private] => -1\n
+)\n
+"""
+EODUMP;
+
+        $this->assertDumpEquals($expected, print_r($clone, true));
```

Comment:
this is playing around assertDumpEquals
just `$this->assertDumpEquals($expected, gmp_init('0101'));` instead